src/Nox.Ui.Blazor.Lib/Models/GenericEntityModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Nox.Types;
+namespace Nox.Ui.Blazor.Lib.Models;
+
+public class GenericEntityModel
+{
+    public GenericEntityModel()
+    {
+    }
+
+    public GenericEntityModel(string? id, string? name)
+    {
+        Id = id;
+        Name = name;
+    }
+
+    public string? Id { get; set; }
```

Comment:
I think we should not be able to pass null values here

Reply:
within latest commit